### ISSUE TITLE
MRTK_Logo.glb import causes warnings

### ISSUE THREAD

**Issue by Alexees** [microsoft/MixedRealityToolkit-Unity#7517]
## Describe the bug

Importing that file, Unity throws two warnings:

1.

```
Unsupported Extension: MSFT_texture_dds
UnityEngine.Debug:LogWarning(Object)
Microsoft.MixedReality.Toolkit.Utilities.Gltf.Serialization.GltfUtility:GetGltfObjectFromJson(String) (at Assets/MixedRealityToolkit/MixedRealityToolkit/Utilities/Gltf/Serialization/GltfUtility.cs:179)
Microsoft.MixedReality.Toolkit.Utilities.Gltf.Serialization.GltfUtility:GetGltfObjectFromGlb(Byte[]) (at Assets/MixedRealityToolkit/MixedRealityToolkit/Utilities/Gltf/Serialization/GltfUtility.cs:252)
Microsoft.MixedReality.Toolkit.Utilities.Gltf.Serialization.<ImportGltfObjectFromPathAsync>d__4:MoveNext() (at Assets/MixedRealityToolkit/MixedRealityToolkit/Utilities/Gltf/Serialization/GltfUtility.cs:122)
System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1:Start(<ImportGltfObjectFromPathAsync>d__4&)
Microsoft.MixedReality.Toolkit.Utilities.Gltf.Serialization.GltfUtility:ImportGltfObjectFromPathAsync(String)
Microsoft.MixedReality.Toolkit.Utilities.Gltf.Serialization.Editor.<OnImportGltfAsset>d__0:MoveNext() (at Assets/MixedRealityToolkit/MixedRealityToolkit/Utilities/Gltf/Serialization/Importers/GltfEditorImporter.cs:16)
System.Runtime.CompilerServices.AsyncVoidMethodBuilder:Start(<OnImportGltfAsset>d__0&)
Microsoft.MixedReality.Toolkit.Utilities.Gltf.Serialization.Editor.GltfEditorImporter:OnImportGltfAsset(AssetImportContext)
Microsoft.MixedReality.Toolkit.Utilities.Gltf.Serialization.Editor.GlbAssetImporter:OnImportAsset(AssetImportContext) (at Assets/MixedRealityToolkit/MixedRealityToolkit/Utilities/Gltf/Serialization/Importers/GlbAssetImporter.cs:13)
UnityEditor.Experimental.AssetImporters.ScriptedImporter:GenerateAssetData(AssetImportContext) (at C:/buildslave/unity/build/Modules/AssetPipelineEditor/Public/ScriptedImporter.cs:20)
```

2.

```
Deprecation warning: Please use an AssetLoaderOptions instance when loading your model, and set it's UseOriginalPositionRotationAndScale field to true.
This new field ensures your model will use the original model local position, rotation and scale, by adding a wrapper on top of the loaded GameObject.
UnityEngine.Debug:LogWarning(Object)
TriLib.AssetLoaderBase:BuildGameObject(AssetLoaderOptions, String, GameObject) (at Assets/TriLib/TriLib/Scripts/AssetLoader/AssetLoaderBase.cs:980)
TriLib.AssetLoader:LoadFromFile(String, AssetLoaderOptions, GameObject, String, ProgressCallback) (at Assets/TriLib/TriLib/Scripts/AssetLoader/AssetLoader.cs:41)
TriLibEditor.TriLibAssetImporter:Import(String) (at Assets/TriLib/TriLib/Editor/TriLibAssetImporter.cs:60)
TriLibEditor.TriLibAssetPostProcessor:CheckForAssimpAsset(String) (at Assets/TriLib/TriLib/Editor/TriLibAssetPostProcessor.cs:48)
TriLibEditor.TriLibAssetPostProcessor:OnPostprocessAllAssets(String[], String[], String[], String[]) (at Assets/TriLib/TriLib/Editor/TriLibAssetPostProcessor.cs:20)
UnityEditor.AssetPostprocessingInternal:PostprocessAllAssets(String[], String[], String[], String[], String[]) (at C:/buildslave/unity/build/Editor/Mono/AssetPostprocessor.cs:141)
```

## To reproduce

1. Reimport MRTK_Logo.glb with a clean console

## Expected behavior

No warnings

## Screenshots

![image](https://user-images.githubusercontent.com/32508825/76393613-308b1980-6374-11ea-9f7b-389988014ebe.png)

## Your setup (please complete the following information)

- Unity Version 2018.3.15f1
- MRTK Version mrtk_development

## Target platform (please complete the following information)

- Editor

**Comment by davidkline-ms** [comment]
@cre8ivepark, can you investigate this import warning? is it related to the process required to render as a 3d tile?

**Comment by keveleigh** [comment]
That first one is a known thing, that the MRTK's glTF loader doesn't support that extension. It's used when the icon is used as a [3D app launcher](https://docs.microsoft.com/en-us/windows/mixed-reality/implementing-3d-app-launchers).

That second one seems unrelated to MRTK. The callstack comes from `TriLib`, and there's maybe a setting you have to set as far as their AssetLoaderOptions config?

**Comment by polar-kev** [comment]
@davidkline-ms, it would be worth our while to update our MRTK icon to not require the extension in question.

**Comment by polar-kev** [comment]
Closing since this was addressed by #9716
